src/main/java/com/example/demo/repository/ProductRepository.java

```diff
@@ -0,0 +1,34 @@
+package com.example.demo.repository;
+
+import com.example.demo.model.Product;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class ProductRepository {
+    private List<Product> products;
+
+    public ProductRepository() {
+    products = new ArrayList<>();
+    products.add(new Product("Komputer", 2000));
+    products.add(new Product("Samochod", 120000));
+    products.add(new Product("Zegarek", 600));
+
+    }
+
+    public void add(Product product) {
+        products.add(product);
+    }
+
+    public ArrayList<Product> listAll() {
```

Comment:
Czemu zwracasz z metody konkretną listę (ArrayList) jest to szczegół który nie jest istnotny dla użytkownika Twoje kodu.